src/main/java/seedu/address/model/person/Doctor.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.person;
+
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.affiliation.Affiliation;
+
+/**
+ * Represents a Doctor in the address book.
+ */
+public class Doctor extends Person {
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Doctor(Name name, Phone phone, Email email, Address address, Set<Affiliation> affiliations) {
```

Comment:
Perhaps we can update address to role as per latest PR?

Reply:
Yup will do so!